src/components/EntityInfo/EntityHeader.vue

```diff
@@ -0,0 +1,105 @@
+<template>
+  <header id="entity-header">
+    <template v-if="!!businessId">
+      <!-- Title -->
+      <div id="entity-legal-name" aria-label="Entity Legal Name">
+        {{ getEntityName || 'Unknown Name' }}
+      </div>
+
+      <!-- Subtitle -->
+      <div>
+        <span id="business-description">{{ businessDescription }}</span>
+
+        <span id="limited-restoration" class="ml-3" v-if="isInLimitedRestoration">
+          <span class="ml-3">Active until {{ getLimitedRestorationActiveUntil || 'Unknown' }}</span>
+          <v-chip class="primary mt-n1 ml-3 pointer-events-none font-weight-bold"
+            small label text-color="white">LIMITED RESTORATION</v-chip>
+        </span>
+
+        <span id="authorized-to-continue-out" v-if="isAuthorizedToContinueOut">
+          <v-chip class="primary mt-n1 ml-3 pointer-events-none font-weight-bold"
+            small label text-color="white">AUTHORIZED TO CONTINUE OUT</v-chip>
+        </span>
+      </div>
+    </template>
```

Comment:
I know this may seem like a pain, but can we separate the code for entities with a `businessId` from entities with a `tempRegNumber` into their own components?  I eventually would like to have separate dashboard routes for entities with a `businessID` and entities with a `tempRegNumber`.  Separating these concerns will make this easier.

Reply:
I'd rather not, at this time anyway. Separate routes sound a good idea for the future, but it's still fair to have separate views use common components that check a variable to display A vs B. Also, I like the fact that the Entity Info's header content is all in one file. Also, I don't want to assume that businesses vs pre-businesses (temp reg number) will continue to be handled separately in the current way. Also, this isn't a complex component to break apart in the future if the need arises (the need isn't here yet). Finally, there are many higher-priority, higher-gain things to work first/instead.

Here's an idea of what we'll be up against if we try to split up businesses from pre-businesses -- the A vs B code is in a lot of places so that we reuse components.

![image](https://user-images.githubusercontent.com/10002889/227250548-f97ec441-0a3f-4b23-98b2-1c32628442de.png)

![image](https://user-images.githubusercontent.com/10002889/227250710-2e88b655-0c3b-4cdf-9f88-b76df6acc1da.png)